obspy/io/nordic/tests/test_nordic.py

```diff
@@ -430,6 +430,9 @@ def test_read_wavename(self):
         testing_path = os.path.join(self.testing_path, '01-0411-15L.S201309')
         wavefiles = readwavename(testing_path)
         self.assertEqual(len(wavefiles), 1)
+        # Check that read_nordic reads wavname when return_wavnames=True
+        cat, wavefiles = read_nordic(testing_path, return_wavnames=True)
+        self.assertEqual(len(wavefiles[0]), 1)
```

Comment:
Any reason to not test the content of `wavefiles` fully?

Reply:
The only reason is that the function `readwavename` which calls `_readwavename` is generally tested well. The only test that was lacking is the call from `read_nordic` to `_readwavename`. 
The 3 lines above are the simplemost test to avoid the bug that led to this PR, by doing the same test as is done on `readwavname` (lines 430-432). It seemed unnecessary to redo the whole test that happens for `readwavname` when only the way that the function is called changes. 
But you can see that I haven't contributed much yet, so I am very happy to extend the tests further if you think it's better to do that.

Reply:
I mean.. what is in this `wavefiles` variable..? It sounds like it should be a filename or something? But it's only length 1. What I was saying is, why only test `len(wavefiles[0])` and not what is actually stored in there?

Reply:
Agree - it may as well check what the returned filename actually is, although from memory the filename returned might be install-path dependent, but that shouldn't be too hard to test.

Reply:
Oh, and @megies, `wavefiles` is a `list` of `str`. Multiple wavefiles can be stored in the nordic s-file, so a list is returned by default.

Reply:
It's no problem to check the actual content of the list of strings `wavefiles` in this case, as it is the filename without a path. 
I was thinking that it would probably not be necessary to duplicate the tests that are done by calling `_readwavname` through `readwavname`, but I have now fixed this very comparison.

Reply:
If you think its good then it maybe is. If it was already tested somewhere then I don't understand how a bug fix was even necessary :stuck_out_tongue_winking_eye:

Reply:
Well, the only thing that was not tested is the alternative way to call the function `_readwavename`. Its output itself was correct if it was supplied with the right input ;-)